components/delete_modal_trigger.test.tsx

```diff
@@ -71,50 +71,22 @@ describe('components/DeleteModalTrigger', () => {
             />
         );
 
-        wrapper.find(ConfirmModal).first().props().onConfirm();
+        wrapper.find(ConfirmModal).first().props().onConfirm(true);
         expect(onDelete).toHaveBeenCalledTimes(1);
     });
 
-    test('should have called onDelete on enter key down', () => {
```

Comment:
How come these tests were removed?

Reply:
@devinbinnie looks like `onKeyDown` was passed to confirm_modal.tsx but was not being used inside confirm_modal.tsx. Which is why i removed these tests as that handler was never being invoked.

https://github.com/mattermost/mattermost-webapp/blob/master/components/confirm_modal.tsx#L8